From f236a30dc8528b6f114201580f1efdcc1c447d43 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Mon, 9 Mar 2020 13:10:37 +0800
Subject: [PATCH] sqlparse: change shebang to python3

Upstream-Status: Pending

Don't send upstream since upstream still support python2,
we can only make this change after python2 is offcially
dropped.

Signed-off-by: Changqing Li <changqing.li@windriver.com>
Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 sqlparse/__main__.py | 2 +-
 sqlparse/cli.py      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sqlparse/__main__.py b/sqlparse/__main__.py
index 2bf2513..6a3a115 100644
--- a/sqlparse/__main__.py
+++ b/sqlparse/__main__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2009-2020 the sqlparse authors and contributors
 # <see AUTHORS file>
diff --git a/sqlparse/cli.py b/sqlparse/cli.py
index 7a8aacb..9c727e8 100755
--- a/sqlparse/cli.py
+++ b/sqlparse/cli.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2009-2020 the sqlparse authors and contributors
 # <see AUTHORS file>
-- 
2.41.0

